packages/@vue/cli-service/generator/template/src/components/HelloWorld.vue

```diff
@@ -3,7 +3,7 @@
   <div class="hello">
     <h1>{{ msg }}</h1>
     <p>
-      For guide and recipes on how to configure / customize this project,<br>
+      For guides and recipes on how to configure / customize this project,<br>
```

Comment:
The docs site only has one "Guide" secion, so I think singular is fine. Would

> For a guide and recipes ...

be better?